### ISSUE TITLE
Cannot freeze database using console 

### ISSUE THREAD

**Issue by antonzherdev** [orientechnologies/orientdb#6505]
## OrientDB Version, operating system, or hardware.
- v2.0 SNAPSHOT[ ] - .18[ ] .17[ ] .16[ ] .15[ ] .14[ ] .13[ ] .12[ ] .11[ ] .10[ ] .9[ ] .8[ ] .7[ ] .6[ ] .5[ ] .4[ ] .3[ ] .2[ ] .1[ ] .0[ ]
- v2.1 SNAPSHOT[ ] - .16[ ] .15[ ] .14[ ] .13[ ] .12[ ] .11[ ] .10[ ] .9[ ] .8[ ] .7[ ] .6[ ] .5[ ] .4[ ] .3[ ] .2[ ] .1[ ] .0[ ]
- v2.2 SNAPSHOT[ ] - .6[X] .rc1[ ] .beta2[ ] .beta1[ ]

## Operating System
- [X] Linux
- [ ] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name? 

## Expected behavior and actual behavior

I'm connecting to the database using console and trying to freeze it. I need it to make AWS snapshot while the database is frozen. It is able to connect and make queries however **freeze database** gives an error:

```
Error: com.orientechnologies.orient.core.exception.OSecurityAccessException: Wrong user/password to [connect] to the remote OrientDB Server instance
	DB name="authdb"
	DB name="authdb"
```

I expect that database freezing should work fine if it can make queries. I'm connecting with the admin user and I did not make any modifications to the user. Only password has been changed.

## Steps to reproduce the problem

Using console:
1. connect to a database
2. freeze database

```
OrientDB console v.2.2.6 (build 2.2.x@r4b1cc998b36ca2ae21a1679938912a8f545a994a; 2016-07-27 15:34:14+0000) www.orientdb.com
Type 'help' to display all the supported commands.
Installing extensions for GREMLIN language v.2.6.0

orientdb> connect remote:/localhost/authdb admin ***

Connecting to database [remote:/localhost/authdb] with user 'admin'...OK
orientdb {db=authdb}> select from Workspace limit 5                 

+----+-----+---------+-----------------+---------------------------------------------------------------------------+
|#   |@RID |@CLASS   |name             |users                                                                      |
+----+-----+---------+-----------------+---------------------------------------------------------------------------+
|0   |#13:0|Workspace|Test             |[#14:0]                                                                    |
|1   |#13:1|Workspace|Flow2B           |[#14:1,#14:8,#14:11,#14:27,#14:29,#14:56,#14:59,#14:60,#14:77,#14:78(more)]|
|2   |#13:2|Workspace|Seymor Industries|[#14:2]                                                                    |
|3   |#13:3|Workspace|Online Money     |[#14:3]                                                                    |
|4   |#13:4|Workspace|two              |[#14:4]                                                                    |
+----+-----+---------+-----------------+---------------------------------------------------------------------------+

5 item(s) found. Query executed in 0.017 sec(s).
orientdb {db=authdb}> freeze database                               

Error: com.orientechnologies.orient.core.exception.OSecurityAccessException: Wrong user/password to [connect] to the remote OrientDB Server instance
	DB name="authdb"
	DB name="authdb"

orientdb {db=authdb}> 
```

**Comment by antonzherdev** [comment]
It's working fine only when the database and server have the same user name and password because the console util uses the password for database to connect to the server with OServerAdmin. But the credentials can be different and I do not see a way how to freeze database in this case.

**Comment by tglman** [comment]
hi @antonzherdev,

The freeze can be done only by server users, it cannot be done by a normal database user, the fact that you defined a server user with the same username and password of a datatabase user make orient use the server difined user instead of the one defined in the database.

so you should not use the database users for freeze but just server users.

**Comment by antonzherdev** [comment]
Hi @tglman 

I do not mind to use server users but if I connect to the server and it asks me to connect to the database so that to know what database should be frozen. When I connect to a database it starts using the database credentials. I looked at the console util's code and I did not find a way how to do it without having the same password for database and the server. Maybe I missed something.
